datalad/config.py

```diff
@@ -120,6 +159,33 @@ def _parse_gitconfig_dump(dump, cwd=None, multi_value=True):
     return dct, fileset
 
 
+# keep alias with previous name for now
+_parse_gitconfig_dump = parse_gitconfig_dump
+
+
+def _gitcfg_rec_to_keyvalue(rec):
+    """Helper for parse_gitconfig_dump()
+
+    Parameters
+    ----------
+    rec: str
+      Key/value specification string
+
+    Returns
+    -------
+    str, str
+      Parsed key and value. Value could be None.
+    """
+    kv_match = cfg_kv_regex.match(rec)
+    if kv_match:
+        k, v = kv_match.groups()
+    else:
+        # could be just a key without = value, which git treats as True
```

Comment:
This is the branch resulting in an additional test outside, no, @mih?

Reply:
Fixed. thx